macros/anaUltraThreshold.py

```diff
@@ -18,7 +20,10 @@
                   help="Make plots vs channels instead of strips", metavar="channels")
 parser.add_option("-p","--panasonic", action="store_true", dest="PanPin",
                   help="Make plots vs Panasonic pins instead of strips", metavar="PanPin")
-
+parser.add_option("--chConfigKnown", action="store_true", dest="chConfigKnown",
```

Comment:
I think a better option would be to store this info in the data file produced by ultraThreshold.py. It is trivial to add these branches by just copying from ultraScurve.py.